src/sonic-config-engine/sonic-cfggen

```diff
@@ -156,6 +157,7 @@ def main():
     group.add_argument("--var-json", help="print the value of a variable, in json format")
     group.add_argument("--write-to-db", help="write config into configdb", action='store_true')
     group.add_argument("--print-data", help="print all data", action='store_true')
+    group.add_argument("--preset", help="preset template name to generate a sample configuration")
```

Comment:
you add options here, so that user will know what presets you have.

Reply:
Added.